docs/001_develop/03_client-capabilities/008_interaction/select.mdx

```diff
@@ -0,0 +1,317 @@
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>form</td>
+      <td><code>string</code></td>
+      <td>Associates this component with a form. Form `id` needs to be passed. If no Id is provided, then it will be associated with the ancestor form.</td>
+<td>
+
+```typescript
+<rapid-select form="sampleForm">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>length</td>
+      <td><code>number</code></td>
+      <td>The number of options.</td>
+<td>
+
+```typescript
+<rapid-select number=10>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>multiple</td>
+      <td><code>boolean</code></td>
+      <td>Enables the user to select more than one option; this automatically opens the selection and removes the side arrow. **Default:** `false`</td>
+<td>
+
+```typescript
+<rapid-select multiple>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>name</td>
+      <td><code>string</code></td>
+      <td>Gives this component a name.</td>
+<td>
+
+```typescript
+<rapid-select name="mySelect">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>open</td>
+      <td><code>boolean</code></td>
+      <td>Defines whether the list starts opened or not. **Default:** `false`</td>
+<td>
+
+```typescript
+<rapid-select open>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>position</td>
+      <td><code>string</code></td>
+      <td>Places the list **above** or **below** the component; can be `above` or `below`. **Default:** it will try to fit with the page.</td>
+<td>
+
+```typescript
+<rapid-select position="below">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>size</td>
+      <td><code>number</code></td>
+      <td>Defines the display size of the list. For example, if you set `size="2"`, then the list displays two items at a time for the user to scroll through; **Default:** it will try to fit with the page.</td>
+<td>
+
+```typescript
+<rapid-select size=2>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>value</td>
+      <td><code>string</code></td>
+      <td>Sets a value for this component.</td>
+<td>
+
+```typescript
+<rapid-select size=2>
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+:::note
+  If you set the `size`, the list is displayed by default (the user does not need to click to view it). This overrides any setting you make for `open`.
+:::
+
+### Option attributes
+
+In order to use the select component, you need to create a list of options for the user to select from. You create this list using `<rapid-option>`. You can define the following attributes for an `<rapid-option>`:
+
+| Name     | Type      | Description                                                                                  |
+|----------|-----------|----------------------------------------------------------------------------------------------|
+| disabled | `boolean` | Disables an option so that the user cannot interact with it, but the option is still visible |
+| selected | `boolean` | Selects the option, so it turns to the selected mode                                         |
```

Comment:
You have missed talking about `value` attribute here - which would be useful, especially as you use that in your three different code examples at the top